lib/internal/readline.js

```diff
@@ -87,33 +87,33 @@ if (process.binding('config').hasIntl) {
     if (
       code >= 0x1100 && (
         code <= 0x115f ||  // Hangul Jamo
-        0x2329 === code || // LEFT-POINTING ANGLE BRACKET
-        0x232a === code || // RIGHT-POINTING ANGLE BRACKET
+        code === 0x2329 || // LEFT-POINTING ANGLE BRACKET
+        code === 0x232a || // RIGHT-POINTING ANGLE BRACKET
         // CJK Radicals Supplement .. Enclosed CJK Letters and Months
-        (0x2e80 <= code && code <= 0x3247 && code !== 0x303f) ||
```

Comment:
Most of the changes seem good to me but I wonder if this whole block is the one exception? It's basically making it look like `0 <= code <= 100`, which is actually kinda nice readability wise.

Reply:
Do you feel strongly about it? Otherwise I would just go ahead and land it as is.

Reply:
Nope. :)